packages/core/upload/admin/src/utils/tests/downloadFile.test.ts

```diff
@@ -10,7 +10,8 @@ describe('downloadFile', () => {
 
     documentSpy.mockReturnValue({
       click: clickSpy,
-      set href(val) {
+      // @ts-ignore I don't need to change the type of the mocked element
+      set href(val: any) {
```

Comment:
what's the error here? I feel like we could avoid ignoring TS by using unknown and adding a typeof check